input/pagecontent/StructureDefinition-au-erequesting-servicerequest-intro.md

```diff
@@ -0,0 +1,15 @@
+### Usage scenarios
+TBD
+
+### Profile specific implementation guidance
+- `ServiceRequest.code` represents the type of service being requested or ordered, such as a diagnostic test or a procedure. Depending on the context provided by `ServiceRequest.category`, `ServiceRequest.code` **SHOULD** be selected from a corresponding domain-specific value set:
+  - When representing a pathology order (`ServiceRequest.category` is SNOMED CT 108252007 |Laboratory procedure), `ServiceRequest.code`:
+    - **SHOULD** use a code from the preferred <a href="https://www.rcpa.edu.au/fhir/ValueSet/spia-requesting-refset-3">RCPA SPIA Pathology Requesting</a> value set
```

Comment:
Suggest we could rephrase this to be simpler and more active language:
- Pathology orders are represented as:
   - `ServiceRequest.category` = "108252007" (Laboratory procedure)
   - `ServiceRequest.code`:
       - **SHOULD** be selected from the preferred <a href="https://www.rcpa.edu.au/fhir/ValueSet/spia-requesting-refset-3">RCPA SPIA Pathology Requesting</a> value set
       - if a system is unable to provide a code from the preferred <a href="https://www.rcpa.edu.au/fhir/ValueSet/spia-requesting-refset-3">RCPA SPIA Pathology Requesting</a> value set it is recommended to select a suitable code from the SNOMED CT <a href="https://healthterminologies.gov.au/fhir/ValueSet/pathology-procedure-1">Pathology Procedure</a> value set.
       - if a system is unable to provide a code from either the preferred <a href="https://www.rcpa.edu.au/fhir/ValueSet/spia-requesting-refset-3">RCPA SPIA Pathology Requesting</a> value set or the SNOMED CT <a href="https://healthterminologies.gov.au/fhir/ValueSet/pathology-procedure-1">Pathology Procedure</a> value set then it is preferred that a local code is provided over only free text

Reply:
Updated